lib/utils.js

```diff
@@ -385,4 +385,44 @@ module.exports = {
       `,\n${propertyText}`
     );
   },
+
+  /**
+   * Whether the provided node represents an autofixer function.
+   * @param {Node} node
+   * @param {Node[]} contextIdentifiers
+   * @returns {boolean}
+   */
+  isAutoFixerFunction (node, contextIdentifiers) {
+    const parent = node.parent;
+    return ['FunctionExpression', 'ArrowFunctionExpression'].includes(node.type) &&
+      parent.parent.type === 'ObjectExpression' &&
+      parent.parent.parent.type === 'CallExpression' &&
+      contextIdentifiers.has(parent.parent.parent.callee.object) &&
+      parent.parent.parent.callee.property.name === 'report' &&
+      this.getReportInfo(parent.parent.parent.arguments).fix === node;
+  },
+
+  /**
+   * Whether the provided node represents a suggestion fixer function.
+   * @param {Node} node
+   * @param {Node[]} contextIdentifiers
+   * @returns {boolean}
+   */
+  isSuggestionFixerFunction (node, contextIdentifiers) {
+    const parent = node.parent;
+    return (node.type === 'FunctionExpression' || node.type === 'ArrowFunctionExpression') &&
+      parent.type === 'Property' &&
+      parent.key.type === 'Identifier' &&
+      parent.key.name === 'fix' &&
+      parent.parent.type === 'ObjectExpression' &&
+      parent.parent.parent.type === 'ArrayExpression' &&
+      parent.parent.parent.parent.type === 'Property' &&
+      parent.parent.parent.parent.key.type === 'Identifier' &&
+      parent.parent.parent.parent.key.name === 'suggest' &&
+      parent.parent.parent.parent.parent.type === 'ObjectExpression' &&
+      parent.parent.parent.parent.parent.parent.type === 'CallExpression' &&
+      contextIdentifiers.has(parent.parent.parent.parent.parent.parent.callee.object) &&
+      parent.parent.parent.parent.parent.parent.callee.property.name === 'report' &&
+      this.getReportInfo(parent.parent.parent.parent.parent.parent.arguments).suggest === parent.parent.parent;
+  },
```

Comment:
can you add a few tests for it, in `tests/lib/utils.js`?

Reply:
Added tests.